### PR TITLE
Cake3 fix event

### PR DESCRIPTION
**Pull request by kevinquinnyo**

### CODE REVIEW

File: src/Auth/CookieAuthenticate.php [hunk 1]
```diff
@@ -120,10 +121,11 @@ public function authenticate(Request $request, Response $response)
     /**
      * Called from AuthComponent::logout()
      *
+     * @param \Cake\Event\Event The dispatched Auth.logout event.
      * @param array $user User record.
      * @return void
      */
-    public function logout(array $user)
+    public function logout(Event $Event, array $user)
```
Comment (ADmad) [inline src/Auth/CookieAuthenticate.php]: `$Event` => `$event`

File: src/Auth/CookieAuthenticate.php [hunk 2]
```diff
@@ -120,10 +121,11 @@ public function authenticate(Request $request, Response $response)
     /**
      * Called from AuthComponent::logout()
      *
+     * @param \Cake\Event\Event The dispatched Auth.logout event.
```
Comment (ADmad) [inline src/Auth/CookieAuthenticate.php]: Var name missing

Comment (kevinquinnyo) [inline src/Auth/CookieAuthenticate.php]: `@param \Cake\Event\Event $event The dispatched Auth.logout event.`

Yeah i saw that too.  It's in the second commit https://github.com/kevinquinnyo/Authenticate/commit/009f728807f1f512eb59cdd09715c719cf7aca7a

**Comment by ADmad** [comment]
Please squash the commits into 1

**Comment by kevinquinnyo** [comment]
@ADmad Did I do that correctly?

**Comment by ADmad** [comment]
Nope, you added one more :)

**Comment by kevinquinnyo** [comment]
bump @ADmad no rush I'm just trying to clear my inbox.  The last PR should satisfy, yes?

**Comment by ADmad** [comment]
The commits are still not squashed into one as I requested.

**Comment by kevinquinnyo** [comment]
Sorry @ADmad I didn't understand you before.  I hope that I did it right this time.  3rd time's a charm?

**Pull request merged**